auacm/app/static/js/controllers/ScoreboardController.js

```diff
@@ -38,26 +37,26 @@ app.controller('ScoreboardController', ['$scope', '$http', '$routeParams',
         var prevSolved = $scope.teams[0].solved;
         var prevTime = $scope.teams[0].solved;
         $scope.teams[0].rank = rank;
-        for (var i = 1; i < $scope.teams.length; i++) {
-            var team = $scope.teams[i];
-            if (team.solved < prevSolved) {
+        for (var j = 1; j < $scope.teams.length; j++) {
```

Comment:
Yeah, I actually found that exact SO post. I fixed it by, at the top of the function, declaring `var team, i;`. I think that's the cleanest way of doing it.